<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="backet1_ballsLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Всего шаров: </string>
    </property>
   </widget>
   <widget class="QLabel" name="backet1_BlueBallsLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Синих шаров:</string>
    </property>
   </widget>
   <widget class="QLabel" name="backet1_RedBallsLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Красных шаров:</string>
    </property>
   </widget>
   <widget class="QLabel" name="backet1_probBlueLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вероятность достать синий</string>
    </property>
   </widget>
   <widget class="QLabel" name="backet1_probRedLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вероятность достать красный</string>
    </property>
   </widget>
   <widget class="QLabel" name="backet1_lastOp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Последнее действие:</string>
    </property>
   </widget>
   <widget class="QLabel" name="backets_prob2BlueLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вероятность достать 2 синих шара:</string>
    </property>
   </widget>
   <widget class="QLabel" name="backets_prob2RedLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вероятность достать 2 красных шара:</string>
    </property>
   </widget>
   <widget class="QLabel" name="backets_prob1Blue1RedLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вероятность достать 1 красный и 1 синий:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backets_get2BallsButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Извлечь 2 шара из случайной корзины</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="backet1_transfButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Переложить шар в корзину 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="backet1_name">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Корзина 1</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="backet2_transfButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Переложить шар в корзину 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="backet2_ballsLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Всего шаров: </string>
     </property>
    </widget>
    <widget class="QLabel" name="backet2_lastOp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Последнее действие:</string>
     </property>
    </widget>
    <widget class="QLabel" name="backet2_probBlueLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Вероятность достать синий</string>
     </property>
    </widget>
    <widget class="QLabel" name="backet2_RedBallsLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Красных шаров:</string>
     </property>
    </widget>
    <widget class="QLabel" name="backet2_probRedLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Вероятность достать красный</string>
     </property>
    </widget>
    <widget class="QLabel" name="backet2_BlueBallsLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Синих шаров:</string>
     </property>
    </widget>
    <widget class="QLabel" name="backet2_name">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Корзина 2</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="probabilitySum_Label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сумма вероятностей: </string>
    </property>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>backet1_ballsLabel</zorder>
   <zorder>backet1_BlueBallsLabel</zorder>
   <zorder>backet1_RedBallsLabel</zorder>
   <zorder>backet1_probBlueLabel</zorder>
   <zorder>backet1_probRedLabel</zorder>
   <zorder>backet1_lastOp</zorder>
   <zorder>backets_prob2BlueLabel</zorder>
   <zorder>backets_prob2RedLabel</zorder>
   <zorder>backets_prob1Blue1RedLabel</zorder>
   <zorder>backets_get2BallsButton</zorder>
   <zorder>probabilitySum_Label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
